src/drt/src/ta/FlexTA_init.cpp

```diff
@@ -843,6 +847,5 @@ void FlexTAWorker::init()
   initIroutes();
   if (getTAIter() != -1) {
     initCosts();
-    sortIroutes();
```

Comment:
Why is this removed?

Reply:
This was migrated to another place (TA main and main_mt). There is one call for clock nets and another for other nets.